upstream/remote/remote.go

```diff
@@ -159,8 +159,8 @@ func (r *Remote) uploadProfile(j *upstream.UploadJob) error {
 	q := u.Query()
 	q.Set("name", j.Name)
 	// TODO: I think these should be renamed to startTime / endTime
-	q.Set("from", strconv.Itoa(int(j.StartTime.Unix())))
-	q.Set("until", strconv.Itoa(int(j.EndTime.Unix())))
+	q.Set("from", strconv.Itoa(int(j.StartTime.UnixNano())))
+	q.Set("until", strconv.Itoa(int(j.EndTime.UnixNano())))
```

Comment:
I think this might be overflowing int32 for everything paste 1970s

Reply:
Yep, nice catch, fixed!